pages/landing/hipaa/technical-details/index.html

```diff
@@ -53,6 +55,239 @@
     Request early access to receive exclusive benefits and discounts!
     <ul class="check-list check-white" style="text-align: left; display: inline-block;"><li><span class="help-text light" title="" data-toggle="tooltip" data-original-title="You’ll be the first to gain access to the solution once it’s ready"> Pre-release HIPAA-Compliant Infrastructure </span> </li><li><span class="help-text light" title="" data-toggle="tooltip" data-original-title="Early adopters will receive the solution at a major discount!"> Early adopter discount</span> </li><li><span class="help-text light" title="" data-toggle="tooltip" data-original-title="We’ll work with you to ensure the solution meets your organization’s requirements"> Influence the product direction</span> </li><li><span class="help-text light" title="" data-toggle="tooltip" data-original-title="We’ll go through the audit with you, implementing any technical changes indicated by the auditor"> HITECH audit support</span> </li></ul>
   cta: Request early access
+
+infra_how_it_works:
+  - heading: End-to-end, opinionated architecture for AWS
+    image: /assets/img/hipaa/infrastructure-hipaa.png
+    image_side: right
+    body: |
+      The Reference Architecture is an opinionated, battle-tested, best-practices way to assemble the code from the <a href="/infrastructure-as-code-library"/>Infrastructure as Code Library</a>) into an end-to-end tech stack that includes just about everything you need: server cluster, load balancer, database, cache, network topology, monitoring, alerting, CI/CD, secrets management, VPN, and more (check out the <a href="/devops-checklist/">Production Readiness Checklist</a> to see what it takes to go to prod).
+
+  - heading: Customizable
+    image: /assets/img/hipaa/hipaa-sample-app.png
+    image_side: left
+    body: |
+      You can choose from EKS, ECS or EC2, databases such as MySQL, Postgres, SQL Server, to customize the Reference Architecture to your exact specifications.
```

Comment:
This text wasn't supposed to be used verbatim either 😕 

It needs a little copywriting to make it clearer that there are a bunch of options. And the screenshot should make the options clearer too.

Reply:
Updated in 145e26b7